docs/loadbalancer-examples.md

```diff
@@ -77,3 +77,44 @@ You can now access your service via the provisioned load balancer
 ```bash
 curl -i http://51.15.224.149
 ```
+
+## Deploy a loadbalancer with specific IP
+
+Instead of creating a loadbalancer service with an ephemeral IP (which change each time your delete/create the service), you can use a reserved loadbalancer address.
+
+_Warning: LoadBalancer ips are different from Instance ips and are not echangeable._
+
+To reserve a loadbalancer IP through API :
+```bash
+curl -X POST "https://api.scaleway.com/lb/v1/regions/$REGION/ips" -H "X-Auth-Token: $SECRET_KEY" -H "Content-Type: application/json" \
+-d "{\"organization_id\":\"$ORGANIZATION_ID\"} | jq -r .ip_address"
```

Comment:
Stay consistent with the variables defined here: https://www.terraform.io/docs/providers/scaleway/index.html#arguments-reference